python/sglang/srt/layers/triton_attention/extend_attention.py

```diff
@@ -284,12 +285,12 @@ def extend_attention_fwd(
         BLOCK_DPE = 0
     BLOCK_DV = triton.next_power_of_2(Lv)
 
-    if CUDA_CAPABILITY[0] >= 9:
+    if torch.cuda.is_available() and CUDA_CAPABILITY[0] >= 9:
```

Comment:
I am not sure whether `torch.cuda.is_available()` is expensive or not. Please cache the results of it at the beginning of this file

Reply:
Done.